src/modules/edc-demo/components/transfer-history/transfer-history-viewer.component.ts

```diff
@@ -1,16 +1,21 @@
 import {Component, OnInit} from '@angular/core';
 import {MatDialog} from '@angular/material/dialog';
-import {Observable, of} from 'rxjs';
 import {map} from 'rxjs/operators';
 import {
   TransferProcessDto,
   TransferProcessService,
 } from '../../../edc-dmgmt-client';
+import {Fetched} from '../../models/fetched';
 import {
   ConfirmDialogModel,
   ConfirmationDialogComponent,
 } from '../confirmation-dialog/confirmation-dialog.component';
 
+export interface TransferProcessesList {
+  transferProcesses: TransferProcessDto[];
+  numTotalTransferProcesses: number;
+}
```

Comment:
is there filtering in the transfer process page that would justify that we can't just get the numTotalTransferProcesses from "transferProcesses.length"?